src/py21cmfast/inputs.py

```diff
@@ -0,0 +1,614 @@
+        so be wary of memory usage if values are high.
+    ZPRIME_STEP_FACTOR : float
+        Logarithmic redshift step-size used in the z' integral.  Logarithmic dz.
+        Decreasing (closer to unity) increases total simulation time for lightcones,
+        and for Ts calculations.
+    TK_at_Z_HEAT_MAX : float
+        If positive, then overwrite default boundary conditions for the evolution
+        equations with this value. The default is to use the value obtained from RECFAST.
+        See also `XION_at_Z_HEAT_MAX`.
+    XION_at_Z_HEAT_MAX : float
+        If positive, then overwrite default boundary conditions for the evolution
+        equations with this value. The default is to use the value obtained from RECFAST.
+        See also `TK_at_Z_HEAT_MAX`.
+    Pop : int
+        Stellar Population responsible for early heating (2 or 3)
+    Pop2_ion : float
+        Number of ionizing photons per baryon for population 2 stellar species.
+    Pop3_ion : float
+        Number of ionizing photons per baryon for population 3 stellar species.
+    NU_X_BAND_MAX : float
+        This is the upper limit of the soft X-ray band (0.5 - 2 keV) used for normalising
+        the X-ray SED to observational limits set by the X-ray luminosity. Used for performing
+        the heating rate integrals.
+    NU_X_MAX : float
+        An upper limit (must be set beyond `NU_X_BAND_MAX`) for performing the rate integrals.
+        Given the X-ray SED is modelled as a power-law, this removes the potential of divergent
+        behaviour for the heating rates. Chosen purely for numerical convenience though it is
+        motivated by the fact that observed X-ray SEDs apprear to turn-over around 10-100 keV
+        (Lehmer et al. 2013, 2015)
+    NBINS_LF : int
+        Number of bins for the luminosity function calculation.
+    P_CUTOFF : bool
+        Turn on Warm-Dark-matter power suppression.
+    M_WDM : float
+        Mass of WDM particle in keV. Ignored if `P_CUTOFF` is False.
+    g_x : float
+        Degrees of freedom of WDM particles; 1.5 for fermions.
+    OMn : float
+        Relative density of neutrinos in the universe.
+    OMk : float
+        Relative density of curvature.
+    OMr : float
+        Relative density of radiation.
+    OMtot : float
+        Fractional density of the universe with respect to critical density. Set to
+        unity for a flat universe.
+    Y_He : float
+        Helium fraction.
+    wl : float
+        Dark energy equation of state parameter (wl = -1 for vacuum )
+    SHETH_b : float
+        Sheth-Tormen parameter for ellipsoidal collapse (for HMF).
+
+        .. note:: The best fit b and c ST params for these 3D realisations have a redshift,
+                  and a DELTA_R_FACTOR (see ANAL_PARAMS.H) dependence, as shown
+                  in Mesinger+. For converged mass functions at z~5-10, set `DELTA_R_FACTOR=1.1`
+                  and `SHETH_b=0.15` and `SHETH_c~0.05`.
+
```

Comment:
These refer to parameters located in the old 21cmFAST files. These should be updated to their new locations (i.e Globals.h and the globals struct).